### PR TITLE
adds indexer readme

### PR DESCRIPTION
**Pull request by tsop14**
@tomrf1

### CODE REVIEW

File: indexer/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+$ sbt "indexer/run <stage>"
+```
+
+You need to connect to the DB by creating an ssh tunnel before you run the
+project. If you run the script `./reindex.sh` this is done for you. ⚡️
+
+## Running Indexer locally
+
+1. From the resources directory create a conf file: `cp application.conf.tempate
+   application.conf`. You only need to fill in the three missing fields, note the
```
Comment (tomrf1) [inline indexer/README.md]: There are 4 missing fields

Comment (tsop14) [inline indexer/README.md]: 👀

**Comment by shtukas** [comment]
:+1:

**Comment by tomrf1** [comment]
👍

**Pull request merged**